backend/api/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from serializers import BusinessSerializer
+
+from sodapy import Socrata
+
+class BusinessesViewSet(mixins.ListModelMixin,
+                        viewsets.GenericViewSet):
+
+    serializer_class = BusinessSerializer
+
+    def get_queryset(self):
+        client = Socrata("data.cityofnewyork.us", "z9bHXnSMLOVBFTGrELvWCcCBN")
+        businesses = client.get("9w7m-hzhe", content_type="json",
+                                limit="500000",
+                                select="camis, dba, boro, building, street, zipcode, cuisine_description")
```

Comment:
btw worry about the `groupby=camis` if that's breaking things. Seems fairly simple to uniquify after the fact, as you're doing below. More importantly you should about what *other* filters you will need to apply, in this case for a bare `/businesses` call from the FE.